compiler/src/dmd/aggregate.d

```diff
@@ -145,6 +145,7 @@ extern (C++) abstract class AggregateDeclaration : ScopeDsymbol
     DtorDeclaration fieldDtor;  /// function destructing (non-inherited) fields
 
     Expression getRTInfo;   /// pointer to GC info generated by object.RTInfo(this)
+    Scope* rtInfoScope;     /// scope to be used when evaluating getRTInfo
```

Comment:
need to update `aggregate.h` and `frontend.h`

Reply:
fixed